src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -741,6 +741,8 @@ void GcsActorManager::LoadInitialData(const EmptyCallback &done) {
   RAY_LOG(INFO) << "Loading initial data.";
   auto callback = [this,
                    done](const std::unordered_map<ActorID, ActorTableData> &result) {
+    std::unordered_map<ClientID, std::pair<rpc::Address, std::vector<WorkerID>>>
```

Comment:
Why not use `std::unordered_map<ClientID, std::unordered_map<rpc::Address, std::vector<WorkerID>>>`?